gds-api-adapters.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'lrucache', '~> 0.1.1'
   s.add_dependency 'rest-client', '~> 1.8.0'
   s.add_dependency 'rack-cache'
+  s.add_dependency 'uk_postcode', '2.1.0'
```

Comment:
I was about to suggest we just `require` it at the top of the `mapit` file, but it would still need installing somehow. I'll look into if we can do it your way...